spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -407,12 +407,20 @@
       end
     end
 
-    context 'when rescued concat Rails.root' do
+    context 'when rescued `Rails.root`' do
       it 'does not register an offense' do
         expect_no_offenses(<<~'RUBY')
           "#{Rails.root rescue '.'}/config"
         RUBY
       end
     end
+
+    context 'with addition operations inside `Rails.root` interpolation expression' do
```

Comment:
The meaning of "addition operations" is unclear. Could you clarify the context of this spec?

Reply:
I updated the spec's description, hope it's more clear now.